contribution.md

```diff
@@ -0,0 +1,113 @@
+# Contributing
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+[Code of Conduct](#code-of-conduct) 
+
+# Pre-Requirements
+To contribute to our community project,please download and install the following:
+
+1.  [Vagrant](https://www.vagrantup.com/)
+2.  [VirtualBox](https://www.virtualbox.org/)
+3.  [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+
+### Run the project on your local machine
+
+A step by step series that tell you how to get Pro4U up and running
+
+1. Clone the repository
+2. Open command prompt in Pro4U directory
+3. Spin the environment using `vagrant up` 
+4. That's it! your virtual machine is up and running 
+5. Go to http://localhost:8000 in your web browser
```

Comment:
Note that it is also possible to enter the website using http://127.0.0.1:8000/, but I don't find this note mandatory